zombienet_tests/functional/0003-parachains-garbage-candidate.toml

```
@@ -0,0 +1,46 @@
+[settings]
+timeout = 1000
+bootnode = true
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+  max_validators_per_core = 2
+  needed_approvals = 4
```

This should make it less flaky hopefully
```suggestion
  needed_approvals = 2
```